### PR TITLE
Increase Kubelet log verbosity in scalability tests

### PR DESCRIPTION
**Pull request by gmarek**
Ref #19048

cc @wojtek-t @kubernetes/goog-control-plane

### CODE REVIEW

**Comment by gmarek** [comment]
Merging.

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -555,6 +555,8 @@ case ${JOB_NAME} in
     NUM_NODES="100"
     # Reduce logs verbosity
     TEST_CLUSTER_LOG_LEVEL="--v=2"
+    # TODO: Remove when we figure out the reason for ocassional failures #19048
+    KUBELET_TEST_LOG_LEVEL="--v=4"
```
Comment (wojtek-t) [inline hack/jenkins/e2e.sh]: isn't v=4 to verbose in kubelet?

Comment (gmarek) [inline hack/jenkins/e2e.sh]: Do you mean isn't it too verbose? We can run it few times and get some failure data. I don't want to keep it this way for long.

Comment (wojtek-t) [inline hack/jenkins/e2e.sh]: What I meant is that I though that all logs that can give us some value (like pod received from watch) are at verbosity 2. Or am I missing something?

Comment (gmarek) [inline hack/jenkins/e2e.sh]: If they are we're clearly not following our own logging guidelines...

**Pull request merged**